test/ufuzz.js

```diff
@@ -575,7 +575,7 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
             createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
       case 28:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '[' +
-            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+            KEYS[rng(KEYS.length)] + "]) ";
```

Comment:
You won't get any pushback from me if you propose removing `mangleProperties` from `test/ufuzz.json` :ghost: